test/SolidityEndToEndTest.cpp

```diff
@@ -2179,6 +2179,21 @@ BOOST_AUTO_TEST_CASE(event_no_arguments)
 	BOOST_CHECK_EQUAL(m_logs[0].topics[0], dev::sha3(string("Deposit()")));
 }
 
+BOOST_AUTO_TEST_CASE(event_anonymous)
+{
+	char const* sourceCode = R"(
+		contract ClientReceipt {
+			event anonymous Deposit();
```

Comment:
To be more in line with function declaration, I think it is better to put anonymous just in front of the `;`.